collector.py

```diff
@@ -0,0 +1,33 @@
+import twitter
+import pickle
```

Comment:
Good move collecting and storing data using pickling. That can save time/energy instead of repeated API calls